### ISSUE TITLE
Request: Lite 151 version

### ISSUE THREAD

**Issue by tjmitchell1** [jojobear13/shinpokered#275]
**Is your feature request related to bug or a functional problem?**
No.

**Is your feature [Within-Scope](https://github.com/jojobear13/shinpokered/blob/master/README.md#will-not-implement--out-of-scope) of the project?**
I believe so.

**Is your feature request related to a game-play problem?**
N/A.

**Describe the feature you would like**
I, and probably others, really love the simplicity of the original game mixed with the bug fixes you have provided via the "Lite" branch you created. The only thing I would like to add to your "Lite" branch is the ability to get all 150 (or 151) pokemon without trading/gameshark cheats. 

**Describe alternatives you've considered**
I have cloned your repository and made the edits for wild pokemon availability and pokemon evolutions from the master branch directly on the lite branch and compiled my own patch. I just think others might also share my idea and would love if it was an "official" branch. 

**Additional context**
Thank you, and the team, for all of the hard work you guys put in!

**Comment by jojobear13** [comment]
Good idea, but there's a hard-cap on the number of patch files that will be supported. Otherwise the logistics becomes too unwieldy.
